test/ip_tests.jl

```diff
@@ -16,4 +16,13 @@
     @test length(ipp.payload) == 48
 end
 
+@testset "ismulticast" begin
```

Comment:
Why do we construct a whole packet to test a function that operators on IP?

Reply:
True, this is wasteful of cognitive resources. I'll change this to a simpler test.

Reply:
Updated with new tests now.